src/components/Counter.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+class Counter extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      number: 0,
+    };
+    this.increment = this.increment.bind(this);
+    this.decrement = this.decrement.bind(this);
+  }
+  increment() {
```

Comment:
You can use arrow function like
`increment = () => {...}`
Then you can get rid of bind(this) in constructor